counterparty-core/counterpartycore/lib/blocks.py

```diff
@@ -1068,6 +1059,7 @@ def parse_new_block(db, decoded_block, block_parser=None, tx_index=None):
     decoded_block["block_index"] = util.CURRENT_BLOCK_INDEX
 
     with db:  # ensure all the block or nothing
+        logger.info(colored(f"Block {decoded_block['block_index']}", attrs=["bold"]))
```

Comment:
okay I changed my mind! I think the whole line including the timestamp and the log level should be bold here.

Reply:
:) done!